descriptors_string_table.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+
+import sys
+
+strings = [x.strip() for x in sys.stdin.readlines()]
+
+if sys.argv[1] == "--header":
```

Comment:
Python is not my strong suite. I have used argparse in the past, but always struggled with it